VickiFreedomMod--VFM--2.0/src/main/java/me/totalfreedom/totalfreedommod/command/Command_happy.java

```diff
@@ -0,0 +1,31 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import me.totalfreedom.totalfreedommod.rank.Rank;
+import me.totalfreedom.totalfreedommod.util.FUtil;
+import org.apache.commons.lang3.StringUtils;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = Rank.OP, source = SourceType.BOTH)
+@CommandParameters(description = "Happy People Only", usage = "/<command>", aliases = "cry")
+public class Command_happy extends FreedomCommand
+{
+    public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        if (sender.getName().equalsIgnoreCase("Charlotte474747")) 
+        if (sender.getName().equalsIgnoreCase("heroguy42"))
```

Comment:
Change this and the one above to:
`if (sender.getName().eqalsIgnoreCase("Chatlotte474747") || sender.getName().equalsIgnoreCase("heroguy42"))`
There is a more efficient/clean way of doing this but I don't have my IDE open.